frontend/public/components/resource-quota.jsx

```diff
@@ -428,6 +428,7 @@ export const ResourceQuotasPage = connectToFlags(FLAGS.OPENSHIFT)(({ namespace,
       title="Resource Quotas"
       rowFilters={rowFilters}
       mock={mock}
+      showTitle={false}
```

Comment:
```suggestion
      showTitle={showTitle}
```

Then change 387 as follows:
```
export const ResourceQuotasPage = connectToFlags(FLAGS.OPENSHIFT)(
  ({ namespace, flags, mock, showTitle = true }) => {
```